tests/playwright/examples/example_apps.py

```diff
@@ -20,10 +20,15 @@ def get_apps(path: str) -> typing.List[str]:
     for folder in os.listdir(full_path):
         folder_path = os.path.join(full_path, folder)
         if os.path.isdir(folder_path):
-            app_path = os.path.join(folder_path, "app.py")
-            if os.path.isfile(app_path):
-                # Return relative app path
-                app_paths.append(os.path.join(path, folder, "app.py"))
+            folder_files = os.listdir(folder_path)
+            for file in folder_files:
+                if os.path.isdir(os.path.join(folder_path, file)):
+                    continue
+                if not file.endswith(".py"):
+                    continue
+                if file == "app.py" or file.startswith("app-"):
+                    # Return relative app path
+                    app_paths.append(os.path.join(path, folder, file))
```

Comment:
This will now add any variation of `app.py`, `app-core.py`, `app-express.py`, `app-other-stuff-core.py`, etc. to the example tests.

This lets us test our Express and Core example variations, but expands the total apps being tested up to ~160 as of right now, probably over 225+ once all  examples are translated.

Reply:
Is this the best testing strategy for express? I feel like testing both versions of each app is a bit overkill since it's basically running the same app twice.

Reply:
Yeah we want to test all the apps that we're using as examples